### DISCUSSION TITLE
PHP 8 support

### DISCUSSION THREAD

**Discussion by Okipa** [deployphp/deployer#2309] [Help needed]
Hello there,

I've made a quick research and I didn't found any information about this.

Right now, I'm stuck upgrading to PHP 8 projects that are using Deployer. I was wondering if adding PHP 8 support was a current work in progress ?

I'm quite interested about having news about this.

Thanks for all your great work.

**Comment by antonmedv** [comment]
Try v7. It supports php8.

**Reply by Okipa** [reply]
Thanks, will try this !
